hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesCapacitySched.java

```diff
@@ -389,4 +384,22 @@ public static void updateTestDataAutomatically(String configFilename, String act
       Assert.fail("overwrite should not fail " + e.getMessage());
     }
   }
+
+  public static WebAppDescriptor createWebAppDescriptor() {
+    return new WebAppDescriptor.Builder(
+        "org.apache.hadoop.yarn.server.resourcemanager.webapp")
```

Comment:
This looks like a volatile constant here. We could use package name of the class here.